src/frontend/context/LapTimesStore/LapTimesStore.tsx

```diff
@@ -5,6 +5,7 @@ export interface LapTimeBuffer {
   lastLapTimes: number[];
   lastSessionTime: number;
   lapTimeHistory: number[][]; // [carIdx][sample]
+  version: number; // Incremented when lapTimeHistory changes
 }
 
 interface LapTimesState {
```

Comment:
I think we might need to add something to reset the state when the session ID changes. I don't remember if its been implemented